task_14/test/home.js

```diff
@@ -0,0 +1,55 @@
+const homepage = require('../pageobjects/homepage')
+var baseUrl = 'https://pastebin.com/'
+
+describe('I Can Win', function(){
+    this.timeout(5000)
+    it('Open https://pastebin.com', function(){
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887247)
```suggestion
    it("Open https://pastebin.com", function(){
```